packages/jaeger-ui-components/src/TracePageHeader/TracePageSearchBar.tsx

```diff
@@ -95,14 +95,17 @@ export default memo(function TracePageSearchBar(props: TracePageSearchBarProps)
   const styles = useStyles2(getStyles);
 
   const suffix = searchValue ? (
-    <span className={styles.TracePageSearchBarSuffix} data-testid="trace-page-search-bar-suffix">
+    <span
+      className={styles.TracePageSearchBarSuffix}
+      data-testid={selectors.components.TraceViewer.tracePageSearchBarSuffix}
+    >
       {searchBarSuffix}
     </span>
   ) : null;
 
   const btnClass = cx(styles.TracePageSearchBarBtn, { [styles.TracePageSearchBarBtnDisabled]: !searchValue });
   const uiFindInputInputProps = {
-    'data-test': markers.IN_TRACE_SEARCH,
+    'data-test': selectors.components.TraceViewer.tracePageSearchBar,
```

Comment:
don't quite understand what this is doing/why we need this? 🤔

Reply:
Removed!